.github/workflows/persistence-performance.yml

```diff
@@ -17,10 +18,13 @@ jobs:
     - uses: actions/checkout@v3
     - name: Set up Python
       uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
     - name: Install requirements
       run: |
         pip install .[tests]
         pip --version
         pip list
     - name: Run persistence performance checks
-      run: python scripts/check_persistence_performance.py
+      run: |
+        python3.11 scripts/check_persistence_performance.py
```

Comment:
this probably can be `python3` or even `python`, right? You can probably revert these two lines to what it was before actually.